lib/BloqadeSchema/src/execute.jl

```diff
@@ -3,25 +3,29 @@
 
 
 
-# """
-#     execute(j::String)
-
-# Executes a task given as a JSON string in the task specification API format, and returns a JSON string of the result
-# """
-# function execute(j::String)
-#     task = Configurations.from_dict(BloqadeSchema.TaskSpecification, JSON.parse(j))
-#     h = from_json(j)
-#     return JSON.json(Configurations.to_dict(execute(h, length(content(h[1]).atoms), 3e-6, task.nshots)))
-# end
-
-# function execute(h::Add, n_atoms::Int, total_time::Float64, nshots::Int)
-#     # Always start off with the zero state
-#     reg = zero_state(n_atoms)
-#     problem = SchrodingerProblem(reg, total_time, h)
-#     emulate!(problem)
-#     bitstrings = reg |> measure(; nshots=nshots)
-#     return to_task_output(bitstrings)
-# end
+"""
+    execute(j::String)
+
+Executes a task given as a JSON string in the task specification API format, and returns a JSON string of the result
+"""
+
+function execute(j::String)
```

Comment:
I think we probably should have three overloads of `execute` for convenience:

```julia
execute(json::Dict)
execute(json::String)
execute(schema::TaskSepcification)
```

since sometimes the schema is already parsed (e.g someone tweaking an existing schema via a `Dict` or `TaskSpecification`). And I think the dispatch here would also be strictly static, so it should be fine.

Reply:
sounds good, I was thinking the same thing.